CHANGES.txt

```diff
@@ -17,6 +17,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       default compiler to MSVC which wasn't installed, yielding broken build.
       Updated mingw tool so that the generate and exists methods use the same mingw search paths
       (issue #4134).
+    - Update the internal mscommon debug handling for stdout to use the logging framework.  Update
```

Comment:
Perhaps change to: 

Update the debug output for MSVC initialization which is enabled by setting SCONS_MSCOMMON_DEBUG to use the logging module. Also changed the debug output format to include more information about the source of each piece of MSVC initialization debugging output.

Also please add same blurb to RELEASE.txt

Reply:
Reworked CHANGES.txt and added same to RELEASE.txt.

Attempt to make clearer that primary change was to debugging output written to stdout.  There was a minor change (adding a space before the message) that affects debugging output written to stdout and files..